app/models/concerns/audit_extensions.rb

```diff
@@ -10,6 +10,17 @@ module AuditExtensions
     before_save :filter_passwords, :if => Proc.new {|audit| audit.audited_changes.try(:has_key?, 'password')}
     after_create :log_audit
 
+    scope :untaxed, -> { by_auditable_types(untaxable_auditables) }
+    scope :taxed_only_by_location, -> { by_auditable_types(location_taxed_auditables) }
+    scope :taxed_only_by_organization, -> { by_auditable_types(organization_taxed_auditables) }
+    scope :fully_taxable_auditables, -> { by_auditable_types(fully_taxable_auditable_types) }
+    scope :by_auditable_types, ->(auditable_types) { where(:auditable_type => auditable_types.map(&:to_s)).readonly(false) }
+    scope :taxed_and_untaxed, lambda {
+      untaxed.or(with_taxonomy_scope { Audit.fully_taxable_auditables })
+             .or(taxed_only_by_organization)
```

Comment:
does this now list all resources that are only taxable by organization, without any restrictions on current organization (the same for location)